src/renderer/components/+namespaces/namespace-select-filter.scss

```diff
@@ -1,5 +1,11 @@
 .NamespaceSelectFilter {
+  --gradientColor: var(--contentColor);
+
   .Select {
+    &.theme-light {
+      --gradientColor: #fff;
```

Comment:
Well, we specifying `#fff` for a theme which is always will be light. If we would set some variable in theme files, we have to write something like `"themeLightNamespaceGradient": #fff` in both `lens-dark.json` and `lens-light.json` files.

Reply:
Yeah, it just seems odd that there is a variable that can be used when it is not light themed, but not when it is light themed. And what if one day a pixel-perfect-minded designer decides the light theme should be more off-white, and this value should be `#efe`? :)

Reply:
@jim-docker That is why I don't particularly think a pure CSS solution is a good idea. Because it basically requires future tweaking. My PR shouldn't need such tweaking in the future.

Reply:
But it should be possible to do it in CSS without requiring future tweaking, right? Variables for foreground and background colours that are a function of the theme? Probably beyond the scope of this PR.

Reply:
Did we consider instead using an ellipsis at front and back of the displayed text as needed?

Reply:
I meant use ellipsis as indicators but still bet able to scroll. Does `text-overflow: ellipsis` do that?

Reply:
No, it truncates output you would have to do the truncation yourself. Which is even more difficult because then you have to care about font rendering.

Reply:
little arrowheads (like &#9658;)? Then you can click on them to scroll